ui/src/app/pages/runtimes-list.tsx

```diff
@@ -34,6 +51,32 @@ const ajax = (signal) => {
   };
 };
 
+const css =
+  hiddenTableColumn.map(({ tableName, columnsToHide }) =>
+    columnsToHide
+      .map(
+        (column) =>
+          `div[aria-label="` +
+          tableName +
+          `"] div[role="columnheader"]:nth-child(` +
+          column +
```

Comment:
`nth-child` takes an index, so you want that to be your variable name here. Otherwise, it looks like a bug.